hdfscm/hdfsmanager.py

```diff
@@ -1,18 +1,21 @@
 import mimetypes
+import os
 from base64 import encodebytes, decodebytes
 from getpass import getuser
-from urllib.parse import urlsplit
+from typing import List
 
 import nbformat
-from notebook.services.contents.manager import ContentsManager
-from pyarrow import hdfs, ArrowIOError
-from traitlets import Unicode, Integer, Bool, default
+if os.getenv('JUPYTER_ENV') == 'dev':
+    from notebook.services.contents.manager import ContentsManager
+else:
+    from jupyter_server.services.contents.manager import ContentsManager
```

Comment:
The test methods still require the notebook dependencies (they don't exist in `jupyter_server` package).